backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1195,7 +1206,7 @@ void convertToDiagnosticReport_Strings_valid() {
 
   @Test
   void convertToDiagnosticReport_Strings_null() {
-    var actual = fhirConverter.convertToDiagnosticReport(null, null, null, null, null);
+    var actual = fhirConverter.convertToDiagnosticReport(null, null, "", null, null, null);
```

Comment:
Should this be an empty string or null?

Reply:
I don't know that it matters too terribly much.  `null` might cover a little more ground testing wise and is more consistent with what is already there, so I'll update it.